### PR TITLE
Adding Sam.gov implementations

### PR DESCRIPTION
**Pull request by brunerae**
Adding Sam.gov implementations.

Sam-Styles - https://github.com/GSA/sam-styles - HTML/CSS design library built on USWDS 2.0 and [Fractal](https://github.com/frctl).
Sam-Design-System - https://github.com/GSA/sam-design-system - Common Angular components integrated with sam-styles css framework.
Sam-Prototypes - https://github.com/GSA/sam-prototypes - A test application that incorporates sam-styles and sam-design-system components with real-world examples.

UI components: For pull requests that impact the look, feel, or functionality of USWDS itself, please open a pull request on the web-design-standards repo (https://github.com/uswds/uswds-site). 

-->

## Description

Include a high-level description of the issue your pull request fixes; include only one issue per pull request. Please also link to the GitHub issue it resolves.

## Additional information

Include any of the following (as necessary): 

* Relevant research and support documents
* Type of content review needed: stylistic or copy editing
* Screenshot images
* Notes
* etc.

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by amyleadem** [comment]
@brunerae 
[Excess spacing before  `- name`](https://github.com/uswds/uswds-site/pull/1714/commits/4be3000935ad48f42260a03527d4edfb5223cd2a#diff-4eb86e0a7715f9b780f49e16707bd05bacd1f9e3809b370eb2100cdd6d7e585aR179) was causing a Jekyll error. I pushed up a change that removes the extra spacing and it resolved the error. 

While I was in there, I also reordered the items to be alphabetical by framework. 

 Updates found in [e2b4586](https://github.com/uswds/uswds-site/pull/1714/commits/e2b45865628b2b39cd503052cc172b3b31241e58).
 
We still have some CircleCI errors, but those should be resolved once https://github.com/uswds/uswds-site/pull/1713 is merged.

File: _data/implementations.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+    name: SAM.gov
+    url: https://github.com/GSA
+  notes: "A test application that incorporates sam-styles and sam-design-system components with real-world examples"
+
+- name: Sam-Design-System
+  distribution: Angular
+  url: https://github.com/GSA/sam-design-system
+  author:
+    name: SAM.gov
+    url: https://github.com/GSA
```
Comment (amyleadem) [inline _data/implementations.yml:14]: I was thrown off by the the `SAM.gov` link pointing to GSA's GitHub. Does it make sense to but the author name as GSA? Or maybe something like SAM.gov/GSA?

Comment (mejiaj) [reply _data/implementations.yml:14]: Probably GSA if it's from there.

File: _data/implementations.yml [hunk 2]
```diff
@@ -1,3 +1,19 @@
+- name: Sam-Prototypes
+  distribution: Angular
+  url: https://github.com/GSA/sam-prototypes
+  author:
+    name: SAM.gov
+    url: https://github.com/GSA
+  notes: "A test application that incorporates sam-styles and sam-design-system components with real-world examples"
```
Comment (mejiaj) [inline _data/implementations.yml:7]: Missing period.

**Review by thisisdano — APPROVED** [review]

**Pull request merged**